furusiya/main.py

```diff
@@ -37,16 +39,16 @@ def main_menu():
         if choice == 1:  # load last game
             try:
                 load_game()
-            except:
+            except Exception as e:

```

Comment:
I don't see any value in addin this if we're not doing anything with `e`. Can't we collapse these two `except` clauses together?

Reply:
There's only one `except` clause.
Admittedly, I added `Exception as e` when I was debugging, and forgot to remove it. It's doing no harm, though.

Reply:
My bad, I read this wrong. This is fine with me. 👍